commands/scanandfixrepos.go

```diff
@@ -63,6 +62,7 @@ func (saf *ScanAndFixRepositories) downloadAndRunScanAndFix(repository *utils.Re
 		}
 	}()
 
-	cfp := CreateFixPullRequestsCmd{dryRun: saf.dryRun, dryRunRepoPath: filepath.Join(saf.dryRunRepoPath, repository.RepoName)}
-	return cfp.scanAndFixRepository(repository, branch, client)
+	cfp := CreateFixPullRequestsCmd{dryRun: saf.dryRun, dryRunRepoPath: saf.dryRunRepoPath}
+	err = cfp.scanAndFixRepository(repository, branch, client)
+	return err
```

Comment:
return err --> return

Let's also use `errors.Join` whenever possible inside this function.